machine_spraak_streaming/lambda/disconnect/main.py

```diff
@@ -0,0 +1,10 @@
+
+
+
+
+def lambda_handler(event, context):
+    print(event)
+
+    return { 
+        'statusCode' : 200
+    }
```

Comment:
Question 1:
I believe that I am not understanding what is happening here. Is there something implicit magic happening with api gateway? :thinking: 

What I mean is that there seems to be no "disconnect logic" inside the function.

Reply:
One always has to return 200 in aws websockets (their convention) and the disconnect route should also exist (but don't think it's enforced). Currently, I have no good use-case for it so there is no logic in there. 

But in the future, a useful thing would simply be to return a "disconnected" message to the client or similar. I think that would help with debugging and understanding the state of the connection for all parties in the connection :)